tomls/markets/perps/btc.toml

```diff
@@ -96,7 +96,11 @@ depends = ["invoke.createBtcPerpsMarket"]
 target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "setMaxMarketSize"
-args = ["<%= settings.btcPerpsMarketId %>", "<%= parseEther(settings.perpsBtcMaxMarketSize) %>"]
+args = [
```

Comment:
Separate `setMaxMarketSize` and `setMaxMarketValue` please

See: https://github.com/Synthetixio/synthetix-v3/pull/2028